pages/categories/[slug].js

```diff
@@ -0,0 +1,75 @@
+import { categories } from "@/lib/data-categories";
+import styled from "styled-components";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import Image from "next/image";
+
+export default function CategoryDetail() {
+  const router = useRouter();
+  const { slug } = router.query;
+  const category = categories.find(cat => cat.slug === slug);
+
+  if (!category) {
+    return (
+      <>
+        <StyledSection>
+          <StyledName>Category not found!</StyledName>
+        </StyledSection>
+      </>
+    );
+  }
+
+  return (
+    <>
+      <StyledBackLink>
+        <Link href="/categories">
+          <Image src="/assets/ArrowIcon.svg" alt="Back Link" width={30} height={25} />
+        </Link>
+      </StyledBackLink>
+      <main>
+        <StyledImage
+          src={category.image}
+          width={200}
+          height={200}
+          alt={category.title}
+        />
+        <StyledSection>
+          <StyledName>{category.title}</StyledName>
+          <article>
+            <h3>Description</h3>
+            <p>{category.description}</p>
+          </article>
+        </StyledSection>
+      </main>
+    </>
+  )
+}
+
+const StyledBackLink = styled.nav`
+  position: absolute;
```

Comment:
There is no `position: relative` container for the absolute positioned `nav` or? 🤔  I would recommend to declare a `relative` container for the absolute positioning to have the full control about the positioning context.